internal/backend/db/dbgorm/scheme/scheme.go

```diff
@@ -49,7 +49,7 @@ var Tables = []any{
 }
 
 type Build struct {
-	BuildID   sdktypes.UUID `gorm:"primaryKey;type:uuid"`
+	BuildID   sdktypes.UUID `gorm:"primaryKey;type:uuid;not null"`
```

Comment:
primaryKey is by definition not null, 
you don't need to add it

Reply:
see this
https://github.com/autokitteh/autokitteh/blob/1af8cad26782595c9ba41d18b3f9e7a019980ddf/migrations/sqlite/20240506105938_baseline.sql#L33